index.html

```diff
@@ -233,18 +238,65 @@ <h1>About Myself</h1>
                   </div>
                 </div>
               </li>
+              <hr />
               <li>
                 <div class="skill-card">
                   <div class="s-title"><h2>Skills</h2></div>
                   <div class="s-icon">
                     <i class="fa-sharp fa-solid fa-angle-right"></i>
                   </div>
                 </div>
+                <hr />
               </li>
             </ul>
           </div>
         </div>
       </div>
     </section>
+
+    <!-- Contact Section -->
+
+    <section class="contact">
```

Comment:
Please consider removing the horizontal scroll from your design as it has a bad impact on the user experience. ⚔️